jfrog-cli-core/go.mod

```diff
@@ -0,0 +1,3 @@
+module jfrog-cli-core
```

Comment:
It looks like this file was added to this PR by mistake.
I recommend going over all the files that were changed before creating a PR.